hs_core/tasks.py

```diff
@@ -860,7 +863,7 @@ def create_bag_by_irods(resource_id, create_zip=True):
                     chksum = istorage.checksum(bag_path)
                     res.bag_checksum = chksum
                 res.setAVU("bag_modified", False)
-                return res.bag_url
+                return istorage.signed_url(bag_path)
```

Comment:
@sblack-usu do we need to modify the `nightly_repair_resource_files` task at all?

Reply:
NVM I see that you changed `check_irods_files`
https://github.com/hydroshare/hydroshare/pull/5578/files#diff-e6fd5b9c81f5b092cfb9f7073b3d0155d9552adfaac9af6f35f7b6d1d2b56a40R298-R306